styles/map.css

```diff
@@ -84,14 +84,48 @@
   width: 50px;
   height: 50px;
 
-  font-size: 50px;
+  /*font-size: 50px;*/
 }
 
 .pin i {
   position: absolute;
 
+  display: inline-block;
+  font-family: "Ionicons";
+  font-style: normal; font-weight: normal; font-variant: normal; text-transform: none;
+
```

Comment:
Hab am Anfang gedacht es geht so schneller aber jetzt ZIEH ich mir das <i> element einfach in javascript und änder da die klasse ;)